src/test/accountExporter.spec.js

```diff
@@ -113,5 +114,119 @@ describe("accountExporter", function() {
+      for (var i = 0; i < 20; i++) {
+        trailingCommas.push(",");
+      }
+      nock("https://www.googleapis.com")
+        .post("/identitytoolkit/v3/relyingparty/downloadAccount", {
+          maxResults: 3,
+          targetProjectId: "test-project-id",
+        })
+        .reply(200, {
+          users: userList.slice(0, 3),
+          nextPageToken: "3",
+        })
+        .post("/identitytoolkit/v3/relyingparty/downloadAccount", {
+          maxResults: 3,
+          nextPageToken: "3",
+          targetProjectId: "test-project-id",
+        })
+        .reply(200, {
+          users: userList.slice(3, 6),
+          nextPageToken: "6",
+        })
+        .post("/identitytoolkit/v3/relyingparty/downloadAccount", {
+          maxResults: 3,
+          nextPageToken: "6",
+          targetProjectId: "test-project-id",
+        })
+        .reply(200, {
+          users: userList.slice(6, 7),
+          nextPageToken: "7",
+        })
+        .post("/identitytoolkit/v3/relyingparty/downloadAccount", {
+          maxResults: 3,
+          nextPageToken: "7",
+          targetProjectId: "test-project-id",
+        })
+        .reply(200, {
+          users: [],
+          nextPageToken: "7",
+        });
+
+      var result = serialExportUsers("test-project-id", {
+        format: "csv",
+        batchSize: 3,
+        writeStream: writeStream,
+      });
+      return result.then(function() {
+        expect(spyWrite.callCount).to.eq(userList.length);
+        for (var j = 0; j < userList.length; j++) {
+          var expectedEntry =
+            userList[j].localId +
+            "," +
+            userList[j].email +
+            ",false,,," +
+            userList[j].displayName +
+            trailingCommas.join("");
+          expect(spyWrite.getCall(j).args[0]).to.eq(expectedEntry + "," + os.EOL);
+        }
+      });
+    });
+
+    it("should encapsulate displayNames with commas for csv formats", function() {
+      // Initialize user with comma in display name.
+      var singleUser = {
+        localId: "1",
+        email: "test1@test.org",
+        displayName: "John Tester1, CFA",
+      };
+      var trailingCommas = [];
+      // The remaining empty columns, index 6 to 25.
+      for (var i = 0; i < 20; i++) {
+        trailingCommas.push(",");
+      }
+      nock("https://www.googleapis.com")
+        .post("/identitytoolkit/v3/relyingparty/downloadAccount", {
+          maxResults: 1,
+          targetProjectId: "test-project-id",
+        })
+        .reply(200, {
+          users: [singleUser],
+          nextPageToken: "1",
+        })
+        .post("/identitytoolkit/v3/relyingparty/downloadAccount", {
+          maxResults: 1,
+          nextPageToken: "1",
+          targetProjectId: "test-project-id",
+        })
+        .reply(200, {
+          users: [],
+          nextPageToken: "1",
+        });
+
+      var result = serialExportUsers("test-project-id", {
```

Comment:
These are looking pretty good. I think the last adjustment that should be made is `result` here - it's really not being used. `serialExportUsers` returns a promise, can this return the promise chain starting there?

```javascript
return serialExportUsers("...")
  .then(function () {
    expect...
  });

Reply:
Done.